base/pointer.jl

```diff
@@ -68,7 +68,7 @@ unsafe_convert(::Type{Ptr{T}}, a::AbstractArray{T}) where {T} = error("conversio
 
 # unsafe pointer to array conversions
 """
-    unsafe_wrap(Array, pointer::Ptr{T}, dims, own=false)
+    unsafe_wrap(Array, pointer::Ptr{T}, dims; own = false)
```

Comment:
Please don't change the whitespace around `=` for everything. Omitting it for keyword arguments is standard style in Base.

Reply:
I think it's fine

Reply:
I personally like it too, but adding such a change in this PR is quite unfortunate because it really invites bikeshedding.

Reply:
Yes, if we could keep this to implementation only, that would be nice.

Reply:
I think it's fine. It would have been preferable to do this without style changes and make style changes separately, but @bramtayl has been kind enough to do a large amount of tedious but important work here, so rather than asking him to do more, let's just focus on getting the meaningful content correct here and we can make our coding styles consistent to our hearts content after 1.0 is out.